packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -434,6 +460,11 @@ location /capabilities {
 # Group: Authentication
 # Description: Access Control Service
 location /acs/api/v1 {
+    # Annotation for IAM upstream metrics.
+    set $upstream_tag IAM;
```

Comment:
Don't we want to distinguish `/internal/acs/api/...` from `/acs/api/...`?

Reply:
Same for MesosDNS.

Reply:
The intent is to allocate requests to upstream services. In that sense, a call to `/internal/X` is the same as a call to `/X`. There will be a breakdown by URI, which, with this change, will separate the `internal` calls from the external calls.